plugin/api.go

```diff
@@ -68,6 +68,9 @@ type API interface {
 	// The status parameter can be: "online", "away", "dnd", or "offline".
 	UpdateUserStatus(userId, status string) (*model.Status, *model.AppError)
 
+	// GetUsersInChannel returns a page of users in a channel. Page counting starts at 0. Sorts by status if requested.
```

Comment:
Can we expand this comment to document the accepted sort parameters?
```suggestion
	// GetUsersInChannel returns a page of users in a channel. Page counting starts at 0. 
	// The sortBy parameter can be: "username" or "status"
```